content/drafts/normal-operation-pre-taproot.md

```diff
@@ -0,0 +1,555 @@
+finally the `onion_routing_packet` and optional `blinding_point` both contain 
+data that the recipient will use to determine where next to send the payment.
+
+### :gear: Step 1: Alice -> Bob: `update_add_htlc(A1)`
+
+Ok so let’s say that Alice sends Bob an `update_add_htlc`. Let’s call this
+HTLC `A1` since it is the first one that Alice (`A`) is sending to Bob. If Bob
+is happy with all the fields in the message, then both parties put this HTLC in
+their staging area commitment transaction as follows:
+
+![](/normalChanOp/state-4.png#center)
+
+Note that neither side has actually committed to this HTLC and so if Bob is a
+routing node for this payment, he should not yet send `update_add_HTLC` to the
+next hop in the route until `A1` has been _irrevocably_ committed. An HTLC
+addition or removal is only considered irrevocably committed once both
+parties in the channel have committed to the commitment transaction with or
+without it respectively.
+
+Something that the more simplified diagram fails to show is that Alice's main
+output in the staging commitment transaction (ie, the `to_local` output on her
+commitment transaction and the `to_remote` on Bob's commitment transaction) will 
+now have the value of the added HTLC subtracted (along with the fees to cover 
+the extra output). If the HTLC ends up succeeding then this amount will be added 
+to Bob's output and if it ends up failing, then it will be re-added to Alice's 
+output.
+
+### :gear: Step 2: Alice -> Bob: `update_add_htlc(A2)`
+
+Even if they have not committed to HTLC `A1` yet, that does not stop them from
+adding more changes to the staging area. So Alice is more than welcome to
+propose a new HTLC, `A2`, to Bob:
+
+![](/normalChanOp/state-5.png#center)
+
+### :gear: Step 3: Bob -> Alice `update_add_htlc(B1)`
+
+Similarly, Bob can suggest a change `B1` to Alice:
+
+![](/normalChanOp/state-6.png#center)
+
+## Committing to the current state
+
+At some point, one of the peers will want to make sure that the other peer has
+committed to the latest set of changes and revoke the previous valid state. This
+is done by sending the `commitment_signed` message:
+
+![](/normalChanOp/commitment_signed.png#center)
+
+The `channel_id` once again refers to the channel in question. The `signature`
+is the sender's signature for the remote party’s staging area commitment
+transaction. `num_htlcs` refers to the number of HTLCs that the sender expects
+to be on the remote commitment transaction and this is then followed by 
+`htlc_signatures` which is an array of `num_hltcs` signatures which are the
+sender’s signatures for each of the second-level HTLC transactions that the
+remote party would need to broadcast if they were ever to force close the 
+channel. If you need a reminder about why the second-level HTLC transactions are 
+needed, give this [post][htlc-deep-dive-post] a read.
+
+### :gear: Step 4: Alice -> Bob: `commitment_signed`
+
```

Comment:
B1 won't be on Bob's commit tx, updates that Bob sends will only get added to Bob's after he sends commit_sig and receives revoke + commit_sig back from Alice

Reply:
fixed!